src/app/modules/event-types-out.ts

```diff
@@ -28,5 +28,5 @@ export interface SnackbarMessage {
 }
 export const SNACKBAR_DISPLAY_NOTIFICATION = 'snackbar.display.notification';
 
-// payload is: Array<string> (path id of resource)
+export interface FilesChangedPayload extends Array<string> {}
```

Comment:
Alternatively, since there's nothing in the interface and its just a type alias, you could use `export type FilesChangedPayload = string[];`